linkup-cli/src/commands/health.rs

```diff
@@ -166,17 +166,30 @@ impl BackgroudServices {
     }
 }
 
+fn is_child_of_current_process(process: &sysinfo::Process, current_pid: sysinfo::Pid) -> bool {
+    match process.parent() {
+        Some(ppid) => ppid == current_pid,
+        None => false,
+    }
+}
+
 fn find_potential_orphan_processes(managed_pids: Vec<services::Pid>) -> Vec<OrphanProcess> {
     let env_var_format = Regex::new(r"^[A-Z_][A-Z0-9_]*=.*$").unwrap();
 
     let current_pid = sysinfo::get_current_pid().unwrap();
     let mut orphans = Vec::new();
 
+    let mut seen_commands = std::collections::HashSet::new();
+
     for (pid, process) in services::system().processes() {
         if pid == &current_pid || managed_pids.contains(pid) {
             continue;
         }
 
+        if is_child_of_current_process(process, current_pid) {
+            continue;
+        }
```

Comment:
I think we can just inline this method here

```suggestion
        // Check if the process is a child of the current process (health command).
        if let Some(parent_pid) = process.parent() {
            if parent_pid == current_pid {
                continue;
            }
        }
```

Cool to know:
[if-let chains](https://github.com/rust-lang/rust/pull/132833) will be stable on 2024 edition when it comes out (I think in June), then we will be able to write like this:
```rust
if let Some(parent_pid) = process.parent() && parent_pid == current_pid {
    continue;
}
```